core/src/graph.jl

```diff
@@ -65,7 +65,14 @@ function create_graph(db::DB, config::Config, chunk_sizes::Vector{Int})::MetaGra
         if ismissing(subnetwork_id)
             subnetwork_id = 0
         end
-        edge_metadata = EdgeMetadata(fid, edge_type, subnetwork_id, (id_src, id_dst))
+        edge_metadata = EdgeMetadata(
+            fid,
+            edge_type == EdgeType.flow ? flow_counter + 1 : 0,
+            edge_type,
+            subnetwork_id,
+            (id_src, id_dst),
+            (0, 0),
```

Comment:
Might be good to set to -1, as 0 has a meaning already. Might also be nice to document inline that it is a placeholder which is overwritten later by `x`.